### ISSUE TITLE
[COGNITION]: Actionable links should have visual cues

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#3117]
## Description

[COGNITION]: Actionable links should have visual cues

<img width="788" alt="direct-deposit-edit-links-issue" src="https://user-images.githubusercontent.com/57469/68237941-2cba4580-ffd6-11e9-823c-5627a2f09ed2.png">
Actionable elements must be identified visually, by convention, and by information provided to assistive technologies. The Edit links have a color difference from the text, but the visual cue is not sufficiently clear that it is actionable.

## Point of Contact

Jennifer

## Acceptance Criteria

As a user of VA.gov, I want to open the Direct Deposit information update functionality. These action links should be clearly visible as actionable links. 

## Environment

* Operating System: all
* Browser: all

## Steps to Recreate

1. Log into staging.va.gov with user 1
2. Click on the person menu in the top right corner
3. Click "Profile" from that menu. Your profile view will load.
4. Click the "Direct deposit information" link to jump to the direct deposit section
5. See any of the Edit links next to Bank name, Account number, or Account type to open the DD modal. They are currently coded as buttons, but styled as links, and need to follow the link styling from [https://design.va.gov/design/typography#links](https://design.va.gov/design/typography#links).

**Comment by andaleliz** [comment]
Closing this out since this design has changed and this is a duplicate of #19562
